draft-ietf-quic-load-balancers.md

```diff
@@ -526,30 +526,33 @@ trusted network service. One requires no shared state; the server need only be
 configured to trust a retry service, though this imposes other operational
 constraints. The other requires shared key, but has no such constraints.
 
-Retry services MUST forward all non-Initial QUIC packets, as well as Initial
```

Comment:
Why was this removed?

Reply:
I was going to rewrite this to allow Retry services to drop 0RTT too, and then realized that this line allows unlimited packets from attackers if they are non-initial, non-0RTT.

This seemed bad, but now I see that because short packets could very easily involve a CID or IP address change, there is no way for the service to filter them effectively. I'll restore it.